docs/upgrading.md

```diff
@@ -5,56 +5,16 @@ There are some major changes between the two library. This guide will help you u
 
 ## New vendor
 
-The new organization name is **PHP HTTP**. This suggests that while our main product is the HTTP Adapter project, we are not limited to it. We plan to collect any kind of HTTP related packages written in PHP.
+The new organization name is **PHP HTTP**. This suggests that while our main product is the Httplug project, we are not limited to it. We plan to collect any kind of HTTP related packages written in PHP.
```

Comment:
i realized that this is not about upgrading, but about migrating. i
started a migration guide that certainly could be elaborated some more.